__tests__/config/environmentConfig.spec.ts

```diff
@@ -57,7 +57,7 @@ describe('Environment environmentConfig', () => {
       const mockConfig = {
         PORT: '5000',
         NODE_ENV: 'qa',
-        APPINSIGHTS_INSTRUMENTATIONKEY: 'abc123',
+        APPLICATIONINSIGHTS_CONNECTION_STRING: 'abc123',
```

Comment:
Also update the environment variables section (https://github.com/DEFRA/ncea-frontend/tree/main?tab=readme-ov-file#environment-variables) in readme.md file

Reply:
Now this has been fixed